dcos_installer/async_server.py

```diff
@@ -358,6 +359,12 @@ def start(cli_options):
     global options
     options = cli_options
 
+    log.error('The DC/OS web installer (--web) is deprecated and will be removed in 1.12\n'
```

Comment:
I think we can just say it is deprecated, I am not sure we need to specify when it will get removed